puppetboard/Dockerfile

```diff
@@ -23,15 +22,15 @@ RUN pip install puppetboard=="$PUPPET_BOARD_VERSION" gunicorn=="$GUNICORN_VERSIO
 
 COPY wsgi.py settings.py /var/www/puppetboard/
 
-EXPOSE $PUPPETBOARD_PORT
+EXPOSE 8000
```

Comment:
What's the rationale behind moving this to a hard-code?

Reply:
from above, "hadolint did not like having the EXPOSE as a variable at all (causes error not just failed test). I tried encapsulating it in a string and Docker did not like that."